.travis.yml

```diff
@@ -28,6 +28,7 @@ before_install:
 
 script:
   - "! git grep -I ' $'" # Fail on trailing whitespace in non-binary files
+  - "! git grep -L --name-only 'Copyright 20[0-9]* Google' | grep -v third_party | egrep '\\.(m|h|cc|mm|c)$'"
```

Comment:
You've got a Y2.1K bug in here :-)

Copyright [0-9]{4} pushes that out to Y10K. (Note you'll need `git grep -E` to match.)